docs/api-reference/mode-handlers/overview.md

```diff
@@ -161,6 +161,9 @@ The following options can be provided in the `modeConfig` object:
   * If unit: `x` means the unit used for the spacing would be of `x` unit.
   * Default unit is `centimeters`
 
+* `lock90Degree`
```

Comment:
I like the way deck.gl does [its reference documentation](http://deck.gl/#/documentation/deckgl-api-reference/layers/layer?section=basic-properties). Specify name, type, optional/required, and default. Something like:

```
* `lock90Degree` (Boolean, optional)
  * Default: `false`

If true, all angles will be guaranteed to be 90 degrees.
```

Reply:
@romaxa please update the doc like Clay suggested.